.github/workflows/check-bioc.yml

```
@@ -252,6 +307,29 @@ jobs:
+      - name: Locate the BiocCheck cache
+        run: |
+          path <- tools::R_user_dir("BiocCheck", "cache")
+          dir.create(path, recursive = TRUE, showWarnings = FALSE)
+          cat(
+              "bioccheck_cache=", gsub("\\\\", "/", path), "\n",
+              file = Sys.getenv("GITHUB_ENV"), sep = "", append = TRUE
+          )
+        shell: Rscript {0}
+
+      - name: Cache the BiocCheck deprecated-package list
+        uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)